Zend/tests/bug79514.phpt

```diff
@@ -1,5 +1,9 @@
 --TEST--
 Bug #79514 (Memory leaks while including unexistent file)
+--SKIPIF--
+<?php
+if (getenv("SKIP_SLOW_TESTS")) die("skip slow test");
+?>
```

Comment:
For this test, I think it should be possible to make it non-slow by dropping those `true` parameters and using a much lower iteration count. There shouldn't be a need for 100000 includes to see the leak.

Reply:
Ah, right, see PR #7088.